Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -209,6 +211,17 @@ public final class PackageInfoMapper: PackageInfoMapping {
                     }
                 }
             }
+        // Include dependencies added as binary targets
+        let remoteXcframeworksPath = path.appending(components: ["artifacts", "tuist"])
+        let rootPath = path.parentDirectory.parentDirectory
```

Comment:
Isn't this assuming the `Package.swift` is in `Tuist/` directory? That's not required. We should probably be using the `RootDirectoryLocator` here.